lib/perl/Genome/Site/TGI/Command/ImportDataFromLims.pm

```diff
@@ -131,7 +142,7 @@ sub _resolve_lims_bam_path {
     my $path;
     while (!$path and @data) {
         my $next = shift @data;
-        $path = $next if $next =~ m!^/gscmnt/!;
+        $path = $next if ($next =~ m!^/gscmnt/! and $next !~ m!^/storage./!);
```

Comment:
I see that this is accounting for the new storage path structure, but it seems odd in general to have to loop over the lines in the file. (Not that that's related to this PR in particular...)

Reply:
It sure does!  It's because the first lines in the file are messages from `docker_run.py` instead of the output of our command.  We could probably work around this by `bsub`bing a short shell script that writes to the file instead of using the output redirection built into LSF.  Then we ought to be able to accept the first line of the file and run with it.